observability/cockpit/how-to/manage-external-data-from-cockpit.mdx

```diff
@@ -0,0 +1,50 @@
+---
+meta: How to manage your external data from your Cockpit
+  title: How to manage your external data from your Cockpit
+  description: This page shows you how to manage your external data from your Cockpit
+content:
+  h1: How to manage your external data from your Cockpit
+  paragraph: This page shows you how to manage your external data from your Cockpit
+categories:
+  - observability
+dates:
+  validation: 2022-10-30
+  posted: 2022-10-30
+---
+
+This page shows you how to manage your external data from your [Cockpit](/observability/cockpit/concepts/#cockpit).
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/)
+  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+
+</Message>
+
+
+1. Click **Cockpit** in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
+
+2. [Create a token](/observability/cockpit/how-to/create-token) to be able to manage your external data.
+
+3. Check **Metrics** and **Logs** for the **Push** permission when selecting your token permissions, so your token can write logs and metrics.
+
+    <Lightbox src="scaleway_token_permissions_external_data.webp" alt="" />
+
+    <Message type="important">
+
+     Once you have created your token, your secret key displays only once. Make sure you copy and save it before you exit the page.
+
+    </Message>
+
+4. Configure the Scaleway endpoints and your newly-created token for your applications or resources.
```

Comment:
Configure your applications so they send their metrics and logs to your Cockpit endpoints, using your newly-created token for authentication.